{"author": {}, "body": "[deleted]", "created_utc": 1586243898.0, "distinguished": null, "id": "fmo3d7z", "is_submitter": false, "link_id": "t3_fuy9mr", "parent_id": "t1_fmo2e5p", "score": 1, "stickied": false, "subreddit_id": "t5_2sjnq", "downs": 0, "total_awards_received": 0, "edited": false, "name": "t1_fmo3d7z", "collapsed": false, "permalink": "/r/TechNewsToday/comments/fuy9mr/no_5g_didnt_cause_the_coronavirus_pandemic/fmo3d7z/"}